pkg/webservice/handlers.go

```diff
@@ -289,7 +292,18 @@ func getApplicationJSON(app *objects.Application) *dao.ApplicationDAOInfo {
 			ApplicationID:    alloc.ApplicationID,
 			Partition:        alloc.PartitionName,
 		}
-		allocationInfos = append(allocationInfos, allocInfo)
+		allocationInfo = append(allocationInfo, allocInfo)
+	}
+
+	for _, taskGroup := range placeholders {
+		phInfo := dao.PlaceholderDAOInfo{
+			TaskGroupName: taskGroup.TaskGroupName,
```

Comment:
Need a test to cover this

Reply:
Thanks. Add a test in handlers_test.go. Add a SetPendingAskRepeat function in allocation_ask.go.